app/src/main/java/co/tinode/tindroid/media/SpanFormatter.java

```diff
@@ -94,6 +95,51 @@ public static boolean hasClickableSpans(final Drafty content) {
         return false;
     }
 
+    /**
+     * Determine attachment type for presentation on UI
+     */
+    public static String getAttachmentType(final Drafty content) {
```

Comment:
This returns the type of the first entity which may not be the attachment. The name of the method is misleading.

Reply:
This method appears to be unused. Why is it needed?

Reply:
In the future for presenting different kind of attachments (i. e. video, audio, pdf document and etc)

Reply:
Please keep it simple. Add only the code needed for the feature you are adding.